src/VisualStudio/Core/Def/Packaging/PackageInstallerServiceFactory.cs

```diff
@@ -136,6 +162,79 @@ private void OnSourceProviderSourcesChanged(object sender, EventArgs e)
             PackageSourcesChanged?.Invoke(this, EventArgs.Empty);
         }
 
+        public bool IsCompatible(
```

Comment:
This doesn't factor in project.json, which means that this check will be incorrect for PCL (targeting UWP/.NET 4.6.1/ASP.NET Core and .NET Standard) and for UWP; because NuGet decided to store TFM information (which is different from what's stored in the project!) in project.json, I would expect this to be passed through to NuGet.